src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,85 +24,70 @@ redirects:
   - /docs/alerts-applied-intelligence
 ---
 
-<LandingPageHero>
-  <HeroContent>
-    Together, New Relic Alerts and Applied Intelligence form a flexible and centralized notification system that unlocks the operational potential of New Relic. Alerts is a single tool to manage alert policies and alert conditions, letting you focus on the metrics you care about most.
+Alerts and applied intelligence are a suite of tools and features that proactively notify you when things go wrong in your system. And when they're starting to go wrong.
 
-    Applied Intelligence helps you find, troubleshoot, and resolve problems faster. Specifically, it's a hybrid machine learning engine that reduces alert noise, correlates incidents, automatically detects anomalies, and provides root cause analysis.
+Alerts lets you define the conditions in your system you want to be notified about, letting you focus on the things you care about most.
 
-    <ButtonGroup>
+Applied intelligence is a machine-learning engine that reduces alert noise, correlates incidents, automatically detects anomalies, and provides root cause analysis. Not only that, it enriches your notifications with metadata to give you greater context of what's happening when you are notified.
+
+<ButtonGroup>
```

Comment:
@paperclypse, if you were to take my suggestion about making an explicit link between the sections of the doc and the UI sections, this call to action seems to be in the way. How about moving it above the image and letting the end of the introduction be a sentence (below the image) that explains the correspondence between the UI sections and the doc sections?